daemon/daemon/daemon.go

```diff
@@ -5,12 +5,25 @@ import (
 	"context"
 
 	"github.com/40acres/40swap/daemon/database"
+	"github.com/40acres/40swap/daemon/lib/lightning/lnd"
```

Comment:
I guess we can simplify everything under `lighthning/lnd` package to avoid more nesting

Reply:
are you sure we want this? Many packages are coming to lib afterwards, I am afraid that this could become infinite folders

Reply:
I dont see many pkgs coming, anyway look LND structure, https://github.com/lightningnetwork/lnd/tree/master, it's pretty flat and `golangish`

Reply:
okay, yep that makes sense

Reply:
done